cmd/gitops/get/config/cmd.go

```diff
@@ -0,0 +1,70 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/weaveworks/weave-gitops/cmd/gitops/config"
+	clilogger "github.com/weaveworks/weave-gitops/cmd/gitops/logger"
+
+	gitOpsCLIConfig "github.com/weaveworks/weave-gitops/pkg/config"
+)
+
+func ConfigCommand(opts *config.Options) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "config",
+		Short: "Prints out the CLI configuration for Weave GitOps",
+		Example: `
+# Prints out the CLI configuration for Weave GitOps
+gitops get config`,
+		SilenceUsage:      true,
+		SilenceErrors:     true,
+		RunE:              getConfigCommandRunE(opts),
+		DisableAutoGenTag: true,
+	}
+
+	return cmd
+}
+
+func getConfigCommandRunE(opts *config.Options) func(*cobra.Command, []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		var err error
+
+		log := clilogger.NewCLILogger(os.Stdout)
+
+		log.Actionf("Reading GitOps CLI config ...")
+
+		configPath, err := gitOpsCLIConfig.GetConfigPath(gitOpsCLIConfig.ConfigFileName)
+		if err != nil {
+			return err
+		}
+
+		configFile, err := gitOpsCLIConfig.OpenConfigFile(configPath, false)
+		if err != nil {
+			log.Failuref("Error opening config file: %w at path: %s", err, configPath)
```

Comment:
This needs to be `%v` to wrap the error, instead of `%w` to debug-print the error. (I think the same thing happens in set config)

Right now it prints `✗ Error opening config file: %!w(*fmt.wrapError=&{failed to open config file: open /home/robin.sonefors/.config/weave-gitops-config.json: no such file or directory 0xc0000376e0})` whereas with `%v` it prints properly.

Reply:
Just replaced `%w` with `%v` in config-related files everywhere, I hope it works.

Reply:
OK, re-replaced `%v` with `%w` where appropriate, I hope NOW it works.